tests/cases/fourslash/unusedImports14FS.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+// @Filename: file2.ts
+//// [| import /* 1 */ A /* 2 */, /* 3 */ { x } from './a'; |]

```

Comment:
My preference, in such cases, is to capture the current behavior but add a comment in the test (which will fail if it changes) explaining that the test is descriptive, rather than normative (i.e. the failure can be addressed by updating the baseline).